docs/localnet.md

```diff
@@ -0,0 +1,54 @@
+# Local Testnets
+
+Prerequisites:
+```bash
+make build
+```
+
+## Single node deployment
+
+To run IPC in the local rootnet just perform the following :
+```bash
+cargo make --makefile ./infra/Makefile.toml node
+
+```
+
+It will create three docker containers (cometbft, fendermint, and eth-api).
+
+To stop run the following:
+```bash
+cargo make --makefile ./infra/Makefile.toml node-down
+```
+
+## Local 4-nodes deployment
+To run IPC in the local rootnet with 4 nodes perform the following command :
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet
+
+```
+
+To stop the network:
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet-down
+```
+
+The testnet contains four logical nodes. Each node consists of cometbft, fendermint, and ethapi containers.
+The testnet network is `192.167.10.0/24`.
```

Comment:
How does this IP work?

Reply:
This network is from comet-bft docker-compose file. I decided to use the same network since the docker-compose file. See https://docs.cometbft.com/v0.37/networks/docker-compose

Reply:
This network is accessible for containers only. You can access it from the host OS.

Reply:
Your two sentences seem to contradict each other, or I am misunderstand what you are trying to say :thinking:

Reply:
Sorry. Misprint, as always.
```
This network is accessible for containers only. You can not access it from the host OS.
```